scripts/plugins/storage/postgres/init.sql

```diff
@@ -739,12 +739,21 @@ CREATE TABLE foglamp.backups (
 -- FogLAMP DB version
 CREATE TABLE foglamp.version (id CHAR(10));
 
--- Create the cnfiguration category_children table
+-- Create the configuration category_children table
 CREATE TABLE foglamp.category_children (
        parent	character varying(255)	NOT NULL,
        child	character varying(255)	NOT NULL,
        CONSTRAINT config_children_pkey PRIMARY KEY (parent, child) );
 
+-- Create the asset_tracker table
+CREATE TABLE foglamp.asset_tracker (
+       asset         character(50)          NOT NULL,
```

Comment:
we should have tacking event id as well. not sure it should be int or uuid

Reply:
cc/ @MarkRiddoch

Reply:
@MarkRiddoch  Thoughts?

Reply:
We don't have a need for it currently, but it is probably best practice to have one.

We also don't have any indexes on this table, although it might not be massive I would suggest we maybe have an index on each of asset and service since these are two comment things we will filter with today.